ci/do_ci.sh

```diff
@@ -9,29 +9,31 @@ echo "building using ${NUM_CPUS} CPUs"
 
 if [[ "$1" == "bazel.debug" ]]; then
   echo "bazel debug build with tests..."
-  ${BAZEL} build ${BAZEL_BUILD_OPTIONS} //source/exe:envoy-static
-  ${BAZEL} test ${BAZEL_BUILD_OPTIONS} --test_output=all \
-    --cache_test_results=no //test/...
+  cd "${ENVOY_CONSUMER_SRCDIR}"
```

Comment:
I think we can cover the build/test of just the main project in the coverage run. When we formalize the build/test matrix we can make this explicit. I think it's useful to make sure that we're checking the main project build/test is sane, but it's not worth burning CI on it today at least (when we free up jobs when removing cmake we could consider this).